dexbot/strategies/relative_orders.py

```diff
@@ -538,8 +538,6 @@ def check_orders(self, *args, **kwargs):
                     self.log.debug('Could not found order on the market, it was filled, expired or cancelled')
                     # Write a trade log entry only when we are not using custom expiration because we cannot
```

Comment:
This comment now irrelevant and should be removed too

Reply:
removed.